app.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask, Response
+
+app = Flask(__name__)
+
+
+# Routes
+@app.route("/")
+def root():
+    return app.send_static_file("index.html")
+
+
+@app.route("/upload")
+@app.route("/upload/")
```

Comment:
You can use `strict_slashes = False` here or set them to be false globally.

https://stackoverflow.com/questions/33241050/trailing-slash-triggers-404-in-flask-path-rule